jsonschema/compat.py

```diff
@@ -28,7 +28,7 @@
     )
     from urllib import unquote  # noqa
     from urllib2 import urlopen  # noqa
-    str_types = basestring
+    str_types = basestring,
```

Comment:
Shouldn't need this to be a tuple, do you?

Reply:
Actually no, I didn't need this to change in the end  - at some point it was looking like I did. 

Isn't it odd that str_types is a tuple in PY3 and not the PY2 though?

Reply:
I mean, mostly it's `isinstance` that's the odd thing and accepts either types or tuples of types, and since this is essentially only used for that, it's "fine" to mix those types.

Doesn't matter too much one way or the other though.